bzt/jmx.py

```diff
@@ -792,14 +793,16 @@ def _get_json_extractor(varname, jsonpath, default='NOT_FOUND', from_variable=No
         return element
 
     @staticmethod
-    def _get_json_path_assertion(jsonpath, expected_value, json_validation, expect_null, invert):
+    def _get_json_path_assertion(jsonpath, expected_value, json_validation, expect_null, invert, regexp):
         """
         :type jsonpath: str
         :type expected_value: str
         :type json_validation: bool
         :type expect_null: bool
         :return: lxml.etree.Element
         """
+        if not regexp:
```

Comment:
There is property to write. You don't know how plugin internally handles this and assumption that escaping regexp is enough might be wrong.